db/migrate/20190129141349_adds_fields_to_expa_application.rb

```diff
@@ -0,0 +1,7 @@
+class AddsFieldsToExpaApplication < ActiveRecord::Migration[5.2]
+  def change
+    add_column :expa_applications, :product, :integer
+    add_column :expa_applications, :podio_id, :integer
+    add_column :expa_applications, :tnid, :integer
```

Comment:
I have created opportunity_id in other branch. I have used the names from expa API